packages/sdk-react/src/components/approvalRequests/ApprovalRequestsTable/ApprovalRequestsTable.tsx

```diff
@@ -165,6 +165,76 @@ const ApprovalRequestsTableBase = ({
     }));
   };
 
+  const columns = useMemo<GridColDef[]>(() => {
+    return [
+      {
+        field: 'number',
+        headerName: t(i18n)`Invoice #`,
+        sortable: false,
+        flex: 0.7,
+      },
+      {
+        field: 'counterpart_id',
+        headerName: t(i18n)`Counterpart`,
+        sortable: false,
+        flex: 1,
+        renderCell: (params) => (
+          <CounterpartCell counterpartId={params.value} />
+        ),
+      },
+      {
+        field: 'issued_at',
+        type: 'date',
+        headerName: t(i18n)`Issue date`,
+        sortable: false,
+        flex: 0.7,
+        valueFormatter: (
+          value: components['schemas']['PayableResponseSchema']['issued_at']
+        ) => value && i18n.date(value, DateTimeFormatOptions.EightDigitDate),
+      },
+      {
+        field: 'due_date',
+        type: 'date',
+        headerName: t(i18n)`Due date`,
+        sortable: false,
+        flex: 0.7,
+        valueFormatter: (
+          value: components['schemas']['PayableResponseSchema']['due_date']
+        ) => value && i18n.date(value, DateTimeFormatOptions.EightDigitDate),
+      },
+      {
+        field: 'status',
+        headerName: t(i18n)`Status`,
+        sortable: false,
+        flex: 0.7,
+        renderCell: (params) => (
+          <ApprovalRequestStatusChip status={params.value} />
+        ),
+      },
+      {
+        field: 'amount',
+        headerName: t(i18n)`Amount`,
+        sortable: false,
+        flex: 0.5,
+        valueGetter: (_, row) => {
+          const payable = row;
```

Comment:
**Small Suggestions:**

No need to create the payable variable here. You can directly access and mutate row.amount_to_pay and row.currency without the extra step, simplifying the code and avoiding unnecessary memory allocation.

```ts
valueGetter: (_, row) => {
  return row.amount_to_pay && row.currency
    ? formatCurrencyToDisplay(row.amount_to_pay, row.currency)
    : '';
},
```

Also, consider applying this approach in other valueGetter logic. I’ve noticed multiple instances where variables are allocated without adding extra value

If you’re not performing additional steps with the variable, try mutating directly